components/camera/videosource/webcam.go

```diff
@@ -2,6 +2,7 @@ package videosource
 
 import (
 	"context"
+	"go.viam.com/rdk/resource"
```

Comment:
There are lots of changes in this file but the big change from previous commits is that I'm now using a reconfigurable camera. I refactored a bit for clarity.